### PR TITLE
docs: simplify README

### PR DESCRIPTION
**Pull request by neerajkumar-n**
Updated Readme, made it leaner

## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [x] Documentation
- [ ] CI/CD

## Description

Changed readme content to make it lean and easier for Contributor to consume content

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?

NA

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +101,39 @@ setup, which includes the [scheduler and monitoring services][docker-compose-sch
...
+
+<a href="https://app.hyperswitch.io/">
+  <h2 id="try-a-payment">⚡️ Try a Payment</h2>
+</a>
+
+To quickly experience the ease of Hyperswitch, sign up on the [Hyperswitch Control Center](https://app.hyperswitch.io/) and try a payment. Once you’ve made your first transaction, congratulations—you’ve successfully completed your first payment with Hyperswitch!
+
+Congratulations 🎉 on making your first payment with Hyperswitch. 
+
+<!--
```
Comment (SanchithHegde) [inline README.md:113]: Can we remove these commented code snippets?

**Review by SanchithHegde — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -135,77 +101,23 @@ setup, which includes the [scheduler and monitoring services][docker-compose-sch
...
-1. Get API keys from our [dashboard].
-2. Follow the instructions detailed on our
-   [documentation page][migrate-from-stripe].
-
-[dashboard]: https://app.hyperswitch.io/register
-[migrate-from-stripe]: https://hyperswitch.io/docs/migrateFromStripe
-
-<a href="#Supported-Features">
-  <h2 id="supported-features">✅ Supported Features</h2>
+<a href="https://app.hyperswitch.io/">
```
Comment (gorakhnathy7) [inline README.md:105]: Hey @neerajkumar-n Can we please move this section before the Quick Setup Section.

File: README.md [hunk 3]
```diff
@@ -135,77 +101,23 @@ setup, which includes the [scheduler and monitoring services][docker-compose-sch
...
-  - Expert team to consult and improve business metrics
-
-You can [try the hosted version in our sandbox][dashboard].
+<a href="#community-contributions">
+  <h2 id="community-contributions">✅ Community & Contributions</h2>
+</a>
 
-<!--
-## Documentation
+The community and core team are available in [GitHub Discussions](https://github.com/juspay/hyperswitch/discussions), where you can ask for support, discuss roadmap, and share ideas.
```
[the remark this answers is not in the document]
Comment (neerajkumar-n) [reply README.md:115]: This is already present in the TOC

File: README.md (hunk 2 above)
Comment (neerajkumar-n) [reply README.md:105]: Don't think that would make sense, the updated format focuses more on Contribution optimization. There is again a TOC - from which any Contributor can Navigate to Trying a Payment.

This is experimental content, which I will be updating over the next few weeks to make it optimal

**Pull request merged**